src/utils/columnPinning.ts

```diff
@@ -0,0 +1,29 @@
+import type React from 'react';
+
+import type {NoStrictEntityMods} from '@bem-react/classname';
+import type {Column} from '@tanstack/react-table';
+
+export function getCommonPinningStyles<TData>(column: Column<TData>): React.CSSProperties {
```

Comment:
Let's rename `getCommonPinningStyles` to `getCellStyles` and set other inline-styles here (`minWidth` and `maxWidth`). Because styles are the same in `Cell` and `HeaderCell`.
It also relates to `getCommonPinningClassModes` and after all you'll need to rename this file as well.